### PR TITLE
Job Client: adds support for progress percent and message to the instance

### PR DESCRIPTION
**Pull request by shamsimam**
## Changes proposed in this PR

- adds support for progress percent and message to the instance

## Why are we making these changes?

Allows users of the job client to query the instance for progress message and percent.

### CODE REVIEW

File: jobclient/src/main/java/com/twosigma/cook/jobclient/Instance.java [hunk 1]
```diff
@@ -220,6 +244,14 @@ public Status getStatus() {
             return _status;
         }
 
+        public Integer getProgress() {
```
Comment (WenboZhao) [inline jobclient/src/main/java/com/twosigma/cook/jobclient/Instance.java]: Can we add document something like "return null if progress info is not available etc. "

File: jobclient/src/main/java/com/twosigma/cook/jobclient/Instance.java [hunk 2]
```diff
@@ -220,6 +244,14 @@ public Status getStatus() {
...
+            return _progress;
+        }
+
+        public String getProgressMessage() {
```
Comment (WenboZhao) [inline jobclient/src/main/java/com/twosigma/cook/jobclient/Instance.java]: similar here. Otherwise, LGTM

Comment (shamsimam) [reply jobclient/src/main/java/com/twosigma/cook/jobclient/Instance.java]: Done.

File: jobclient/src/main/java/com/twosigma/cook/jobclient/Instance.java (hunk 1 above)
Comment (shamsimam) [reply jobclient/src/main/java/com/twosigma/cook/jobclient/Instance.java]: Done.

**Pull request merged**